CHANGELOG/CHANGELOG-3.4.md

```diff
@@ -35,6 +35,7 @@ Previous change logs can be found at [CHANGELOG-3.3](https://github.com/etcd-io/
 - Upgrade [bbolt to 1.3.9](https://github.com/etcd-io/etcd/pull/17484).
 - Compile binaries using [go 1.21.8](https://github.com/etcd-io/etcd/pull/17538).
 - Upgrade [google.golang.org/protobuf to v1.33.0 to address CVE-2024-24786](https://github.com/etcd-io/etcd/pull/17554).
+- Fix [PRISMA-2023-0056](https://github.com/etcd-io/etcd/pull/17580).
```

Comment:
Suggestion to specify the dependency bumped.

```suggestion
- Upgrade github.com/sirupsen/logrus to v1.9.3 to address [PRISMA-2023-0056](https://github.com/etcd-io/etcd/pull/17580).
```

Reply:
Update v3.4/3.5 @ivanvc thanks.